circuit_definitions/src/circuit_definitions/aux_layer/mod.rs

```diff
@@ -422,19 +423,19 @@ impl ZkSyncCompressionForWrapperCircuit {
         use crate::boojum::cs::traits::circuit::CircuitBuilder;
         match &self {
             Self::CompressionMode1Circuit(..) => {
-                <CompressionMode1Circuit as CircuitBuilder<GoldilocksField>>::geometry()
+                <CompressionMode1ForWrapperCircuit as CircuitBuilder<GoldilocksField>>::geometry()
```

Comment:
This may break existing functionality, I do not remember the path of how current scheduler proof is wrapper (most likely 1 FRI based proof to switch to BN254 hasher, and then PLONK proof), so such code path may be used and will break.

In general, would be nice to have comment on what's a difference between compression and compression for wrapper, especially at initial steps, as only one step should swap hasher